### PR TITLE
chore(suite): update network symbol naming  10

### PR DESCRIPTION
**Pull request by adderpositive**
## Description
- update network symbol naming 

## Related 
Follow up https://github.com/trezor/trezor-suite/pull/15810

### CODE REVIEW

File: suite-common/wallet-core/src/send/sendFormReducer.ts [hunk 1]
```diff
@@ -138,12 +138,12 @@ export const selectSendFormDraftOutputsByAccountKey = (
...
 export const selectSendFormReviewButtonRequestsCount = (
     state: DeviceRootState,
-    networkSymbol?: NetworkSymbol,
+    symbol?: NetworkSymbol,
     decreaseOutputId?: number,
 ) => {
     const buttonRequestCodes = selectDeviceButtonRequestsCodes(state);
     const deviceModel = selectDeviceModel(state);
-    const { networkType } = networks[networkSymbol ?? 'btc'];
+    const networkType = getNetworkType(symbol ?? 'btc');
```
Comment (tomasklim) [inline suite-common/wallet-core/src/send/sendFormReducer.ts:146]: can we get rid of `btc`?

File: suite-native/discovery/src/discoveryThunks.ts [hunk 2]
```diff
@@ -74,9 +73,9 @@ export const discoveryErrors = {
     accountLimitReached: 'discovery-account-limit',
 };
 
-const getBatchSizeByCoin = (coin: NetworkSymbol): number => {
-    if (coin in DISCOVERY_BATCH_SIZE_PER_COIN) {
-        return DISCOVERY_BATCH_SIZE_PER_COIN[coin]!;
+const getBatchSizeByCoin = (symbol: NetworkSymbol): number => {
```
Comment (tomasklim) [inline suite-native/discovery/src/discoveryThunks.ts:76]:
```suggestion
const getBatchSizeByNetworkSymbol = (symbol: NetworkSymbol): number => {
```

File: suite-native/discovery/src/discoveryThunks.ts [hunk 3]
```diff
@@ -74,9 +73,9 @@ export const discoveryErrors = {
...
+    if (DISCOVERY_BATCH_SIZE_PER_COIN[symbol] !== undefined) {
```
Comment (tomasklim) [inline suite-native/discovery/src/discoveryThunks.ts:77]:
```suggestion
    if (DISCOVERY_BATCH_SIZE_PER_SYMBOL[symbol] !== undefined) {
```

**Review by tomasklim — APPROVED** [review]

**Comment by PeKne** [comment]
This PR was merged with [failing mobile E2E tests](https://github.com/trezor/trezor-suite/actions/runs/12214607953/job/34075691581)📛 and broke mobile send flow. Quickfix prepared: https://github.com/trezor/trezor-suite/pull/15842.

**Pull request merged**